ext/js/pages/settings/dictionary-controller.js

```diff
@@ -775,9 +875,27 @@ export class DictionaryController {
         }
     }
 
+    /** */
+    async _checkForUpdates() {
+        if (this._dictionaries === null || this._checkingIntegrity || this._checkingUpdates || this._isDeleting) { return; }
+        try {
+            this._checkingUpdates = true;
+            this._setButtonsEnabled(false);
+
+            const updateChecks = this._dictionaryEntries.map((entry) => entry.checkForUpdate());
+            const updateCount = (await Promise.all(updateChecks)).reduce((sum, value) => (sum + (value ? 1 : 0)), 0);
+            if (this._checkUpdatesButton !== null) {
+                this._checkUpdatesButton.textContent = updateCount ? `${updateCount} updates` : 'No updates';
+            }
```

Comment:
Nitpick: when there's only one update, the text should be singular rather than plural